kedro/templates/project/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -12,3 +12,4 @@ nbstripout~=0.4
 pytest-cov~=3.0
 pytest-mock>=1.7.1, <2.0
 pytest~=7.2
+traitlets<5.10.0
```

Comment:
Maybe leave a comment noting https://github.com/jupyter/notebook/issues/7048 ?

Reply:
Will do in my other PR, just lazy to re-trigger the CI tests